deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+
+### Uploading Data
+
+Navigate to Orthanc via http://localhost:8042 or http://172.29.0.100:8042, and click *Upload* on the top right-hand corner.
+Drag & drop any DICOM files (no folders) to the blue box on the page and then click *Start the upload*.
+
+Or, upload your files to Orthanc using the *storescu* command from [dcmtk](https://dcmtk.org/dcmtk.php.en).
+```
+storescu -v +r +sd -aec ORTHANC localhost 4242 /path/to/my/dicom/*
+```
+
+Navigate to the home page and click *All studies* to confirm data's been uploaded.
+
+### Sample Workflows
+
+Under the **sample-workflows* directory, a couple of sample workflow definitions are provided:
+
+- `hello-world.json `: Hello World!
+- `lung-seg.json`: AI Lung Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+
+
+#### Hello World
+
+##### Description
+
+This example uses the `alpine` image to print all files found in the input directory simply.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"849d4683-006b-410c-aa17-d0474ee26b7b"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+1. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
+1. To see the output from the container, run the following commands:
+   ```bash
+   $ docker container list -a | grep alpine
+   # locate the container ID and run the following
+   $ docker logs {CONTAINER ID}
+   # expect a list of files to be printed
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm.json
+   /var/monai/input/1.2.826.0.1.3680043.2.1125.1.19616861412188316212577695277886020/1.2.826.0.1.3680043.2.1125.1.34918616334750294149839565085991567/1.2.826.0.1.3680043.2.1125.1.60545822758941849948931508930806372.dcm
+   ...
+   ```
+
+#### AI Liver Segmentation MAP
+
+##### Description
+
+In this section, we will download a DICOM dataset, upload it to Orthanc and then run the [Liver Segmentation MAP](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/ai_livertumor_seg_app) from the
+[MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk). Finally, we can expect the AI-generated segmentation result to appear in Orthanc.
+
+1. Download the dataset from [here](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing)
+2. Upload the dataset as described in [Uploading Data](#uploading-data)
```

Comment:
This is downloading from a Google drive, do users need a Google account to download?

Reply:
@nakfour can you try it without logging on to Google. Download can be set to anyone with the link can access/download.